src/Tribe/Admin/Columns/Tickets.php

```diff
@@ -121,51 +125,32 @@ protected function get_sold( $tickets ) {
 	}
 
 	/**
-	 * Iterates over an array of tickets to render the percentage HTML.
+	 * Gets the HTML for the percentage string for Attendees Column
 	 *
-	 * @param  int $post_id The current post ID.
-	 * @param Tribe__Tickets__Ticket_Object[] $tickets
+	 * @since  TBD  Deprecated the Second Param
+	 *
+	 * @param  int                             $post_id    The current post ID.
+	 * @param  Tribe__Tickets__Ticket_Object[] $deprecated
 	 *
 	 * @return string The percentage HTML or an empty string if one of the
 	 *                post tickets has unlimited stock.
 	 */
-	protected function get_percentage_string( $post_id, array $tickets = array() ) {
-		$sold  = 0;
-		$stock = 0;
-
-		$global_stock_enabled = get_post_meta( $post_id, Tribe__Tickets__Global_Stock::GLOBAL_STOCK_ENABLED, true ) == true;
-		$global_stock = $global_stock_enabled ?
-			get_post_meta( $post_id, Tribe__Tickets__Global_Stock::GLOBAL_STOCK_LEVEL, true )
-			: false;
+	protected function get_percentage_string( $post_id, array $deprecated = array() ) {
```

Comment:
Question: does it make sense to provide a default value, `array()`, for a deprecated value? Would it not be better to default to `null`?